qiskit/transpiler/passes/layout/vf2_utils.py

```diff
@@ -12,31 +12,18 @@
 
 """This module contains common utils for vf2 layout passes."""
 
+__all__ = ["ErrorMap"]
+
 from collections import defaultdict
-import random
 
-import numpy as np
 from rustworkx import PyDiGraph, PyGraph, connected_components, weakly_connected_components
 
+from qiskit._accelerate.error_map import ErrorMap
```

Comment:
Do we need to support exporting `ErrorMap` from `vf2_utils`? I guess the question here if (backward-compatibility issues aside) it would make sense to export `ErrorMap`from the parent `layout` directory instead?